src/tui/mod.rs

```diff
@@ -114,6 +121,7 @@ fn render(tui: &Tui, area: Rect, buffer: &mut Buffer) -> miette::Result<()> {
 
 const SCROLL_AMOUNT: usize = 1;
 
+#[instrument(level = "trace", skip(tui))]
 fn handle_event(tui: &mut Tui, event: Event) -> miette::Result<()> {
     match event {
         // TODO(evan): Scrolling is excruciatingly slow
```

Comment:
Stale todo now that `gag` is gone

Reply:
Oh I thought that referred to the scroll speed rather than the event speed.